ports/atmel-samd/common-hal/pulseio/PulseIn.c

```diff
@@ -298,7 +298,7 @@ void common_hal_pulseio_pulsein_clear(pulseio_pulsein_obj_t* self) {
 
 uint16_t common_hal_pulseio_pulsein_popleft(pulseio_pulsein_obj_t* self) {
     if (self->len == 0) {
-        mp_raise_IndexError(translate("pop from an empty PulseIn"));
+        mp_raise_IndexError_varg(translate("pop from empty %s"), "PulseIn");
```

Comment:
Why string here but qstr below?

Reply:
oversight, will harmonize these to be %q

Reply:
Nice, another +12 bytes free